scrapy/extensions/feedexport.py

```diff
@@ -288,7 +293,7 @@ def close_spider(self, spider):
     def item_scraped(self, item, spider):
         for slot in self.slots:
             slot.start_exporting()
-            slot.exporter.export_item(item)
+            slot.export_item(item)
             slot.itemcount += 1
```

Comment:
This increases itemcount even when item is not exported.

Reply:
Good catch!